Snapshots/Snapshots.cpp

```diff
@@ -421,6 +422,12 @@ void SWS_SnapshotsWnd::Update()
 		CheckDlgButton(m_hwnd, cSSCtrls[i], g_iMask & cSSMasks[i] ? BST_CHECKED : BST_UNCHECKED);
 		EnableWindow(GetDlgItem(m_hwnd,  cSSCtrls[i]), m_iSelType == 2);
```

Comment:
The check below could be done here so that it gets applied to any future version-specific features without code duplication:
```suggestion
		CheckDlgButton(m_hwnd, cSSCtrls[i], g_iMask & cSSMasks[i] & g_compatMask ? BST_CHECKED : BST_UNCHECKED);
		EnableWindow(GetDlgItem(m_hwnd,  cSSCtrls[i]), m_iSelType == 2 && cSSMasks[i] & g_compatMask);
```